lib/run.js

```diff
@@ -50,19 +53,19 @@ function run(config, done) {
                     total: tasks.length,
                     info: info,
                     testCmd: testCmd,
-                    installCmd: "npm i" + (config.registry ? (" --registry " + config.registry) : "")
+                    installCmd: (config.useYarn ? "yarn add" : "npm i") + (config.registry ? (" --registry " + config.registry) : "")
```

Comment:
I am pretty sure yarn can do this, I just did not research how. I will have a look later on.